.github/workflows/pr-if-new-kernel.yml

```diff
@@ -1,7 +1,7 @@
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '0 5 * * *'
```

Comment:
Why changing the time here?

Reply:
Just to get some space between our two automated runs, build first, check for updates later. Does not really matter but feels more logical to me.